project/Dependencies.scala

```diff
@@ -13,4 +13,8 @@ object Dependencies {
     "com.geirsson" %% "coursier-small" % "1.3.3",
     "io.get-coursier" % "interface" % "1.0.4"
   )
+
+  val sbt1Plugins = List(
+    "com.eed3si9n" % "sbt-projectmatrix" % "0.8.0"
```

Comment:
This is adding a new dependency also for scalafix users (not rule authors). It does look like there is a MiMa on it, so this could be a problem if it evicts the user's potential version. I think the risk is very limited, so unless we have such reports, I don't think it's worth creating a separated `sbt-scalafix-testkit` artifact.

Reply:
https://github.com/scalacenter/sbt-scalafix/pull/222